docs/codelabs/01-sql-statement.md

```diff
@@ -48,7 +48,7 @@ CLUSTER`, which operates on cluster settings, `FROBNICATE SESSION`, working on
 session settings, and `FROBNICATE ALL`, which handles both.
 
 Let's start by checking to make sure all our keywords are defined.  Open
-`pkg/sql/parser/sql.y` and search for "keyword".  You'll find a series of token
+`pkg/sql/parser/sql.y` and search for "Ordinary key words".  You'll find a series of token
```

Comment:
The search should be for `unreserved_keyword`

Reply:
This is for adding the new keyword to the section starting line 554 in `sql.y`. Further down in this code lab (line 65) you'll see that we guide people to add specifically to the `unreserved_keyword` list.

Reply:
Oh i see. Looks good then.

Reply:
Thanks for reviewing!